frontend/www/js/omegaup/group/edit.ts

```diff
@@ -251,18 +240,30 @@ OmegaUp.on('ready', () => {
           }) => {
             CSV.fetch({ file })
               .done((dataset: CSV.Dataset) => {
-                if (!dataset.fields || dataset.fields.length != 6) {
+                if (!dataset.fields) {
                   ui.error(T.groupsInvalidCsv);
                   return;
                 }
-                for (const [
+                const records = getCSVRecords<GroupCSVDatasetRecord>({
+                  fields: dataset.fields,
+                  records: dataset.records,
+                  requiredFields: new Set([
+                    'username',
+                    'name',
+                    'country_id',
+                    'state_id',
+                    'gender',
+                    'school_name',
+                  ]),
```

Comment:
```suggestion
                  requiredFields: new Set([
                    'username',
                  ]),
                  optionalFields: new Set([
                    'name',
                    'country_id',
                    'state_id',
                    'gender',
                    'school_name',
                  ]),
```

resulta que el API permite que casi todos los campos sean opcionales, jaja. entonces está bien que esos campos lo sean.

estos `requiredFields` y `optionalFields` se pueden exportar para las pruebas para no repetir estas cosas tanto.

Reply:
Listo.